lib/please_cli/please_cli/config.py

```diff
@@ -65,6 +67,8 @@
 PROJECTS += ['scriptworker/shipitscript']
 PROJECTS += DEV_PROJECTS
 
+LOCAL = LocalConfig()
```

Comment:
I think LOCAL is not really a descriptive name maybe:
```suggestion
CONFIG = LocalConfig()
```